app/src/main/java/org/fossasia/openevent/api/processor/MicrolocationListResponseProcessor.java

```diff
@@ -13,12 +16,29 @@
  */
 public class MicrolocationListResponseProcessor implements Callback<MicrolocationResponseList> {
     private static final String TAG = "Microprocessor";
+    ArrayList<String> queries = new ArrayList<String>();
 
     @Override
     public void success(MicrolocationResponseList microlocationResponseList, Response response) {
+        for (Microlocation microlocation : microlocationResponseList.microlocations)
+
+        {
```

Comment:
I am not so sure, I use auto-formatting everyday in my workflow and don't have situations like these, you should select all the changed files under the changes tab in Android Studio -> Right click -> Reformat code.